yarn.lock

```diff
@@ -417,6 +417,15 @@
     debug "^4.1.0"
     globals "^11.1.0"
 
+"@babel/types@7.21.4", "@babel/types@^7.21.4":
```

Comment:
Quick question: The `@babel/types@^7.21.4` one (with the caret) is a transitive dependency, right? So basically a dependency of ours using that version constraint?

Reply:
I think so, yeah :)